src/content/packages-and-plugins/using-packages.md

```diff
@@ -145,7 +145,7 @@ To add the package, `css_colors`, to an app:
 To remove the package, `css_colors`, to an app:
 
 1. Issue the command while being inside the project directory
-   * `flutter pub remove css_colors`
+   * `flutter pub remove css_colors:'^1.0.0'`
```

Comment:
I don't think `pub remove` accepts a source descriptor, just one or more package names, so no need for this change.

```suggestion
   * `flutter pub remove css_colors`
```